adrs/0001-event-sorcery-persistence-foundation.md

```
@@ -1,3 +1,4 @@
+<<<<<<< New base: chore: adopt the event-sorcery event-store stack (sqlx 0.9, apalis 1.0-rc) (#361
```

markdownlint-cli2 (0.22.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)